stl/debugger/STL.natvis

```diff
@@ -400,6 +400,14 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       </Expand>
   </Type>
 
+  <!-- VS 2019 16.3 -->
+  <Type Name="std::_Ref_count_obj2&lt;*&gt;">
+      <DisplayString>make_shared</DisplayString>
+      <Expand>
+          <Item Condition="_Uses != 0" Name="[original ptr]">($T1 *) &amp;_Storage</Item>

```

Comment:
You know you've been working on the STL too long when your first reaction to `($T1 *) &amp;_Storage` is to worry about ADL hijacking `&amp;`. (No change requested - just sharing.)